modules/hitl/src/views/full/sidebar/index.jsx

```diff
@@ -26,7 +26,19 @@ export default class Sidebar extends React.Component {
     this.props.toggleOnlyPaused()
   }
 
-  renderUser = (value) => {
+  toggleSearch = () => {
+    this.props.customSearchProps.searchClickAction()
+  }
+
+  clearSearch = () => {
+    this.props.customSearchProps.searchClearAction()
+  }
```

Comment:
Instead of simply forwarding your calls, you could call those directly using this.props... Also cleaner if you remove the customSearchprops property

Reply:
[DONE](https://github.com/shamoh19/botpress/commit/3c171a8926b82c779d9f24997da72b05e98cd396#diff-b23a720bad734b651015f6507e7d6c71)